app/_kong_plugins/acme/examples/redis-aws-instance.yaml

```diff
@@ -0,0 +1,71 @@
+title: ACME with Redis storage and AWS ElastiCache instance auth
+description: |
+  Configure the ACME plugin with Redis as a storage backend using AWS ElastiCache instance auth
+
+weight: 860
+
+requirements:
+- A public IP and a resolvable DNS
+- '{{site.base_gateway}} accepts proxy traffic on port 80'
+- A running Redis instance on an [AWS ElastiCache instance](https://docs.aws.amazon.com/AmazonElastiCache/latest/dg/auth-iam.html) for Valkey 7.2 or later or Elasticache for Redis OSS version 7.0 or later
```

Comment:
Shall we also document the necessary IAM permissions here? Same for the other two CSP

Reply:
- For AWS Elasticache: 
1) the Elasticache user needs to set "Authentication mode" to "IAM"
2) the IAM user/IAM role that being used to connect to the Elasticache needs the following policy assigned:

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "elasticache:Connect"
            ],
            "Resource": [
                "arn:aws:elasticache:ARN_OF_THE_ELASTICACHE",
                "arn:aws:elasticache:ARN_OF_THE_ELASTICACHE_USER"
            ]
        }
    ]
}
```